tests/validation_tests/test_pre_checks.py

```diff
@@ -336,3 +342,10 @@ def test_place_name_is_valid(valid_submission_details):
     failures = pre_transformation_check(valid_submission_details)
     assert len(failures) == 1
     assert isinstance(failures[0], vf.WrongInputFailure)
+
+
+def test_unauthorised_submission():
+    unauthorised_failure_dict = {"Unauthorised Place Name": ("Newark", ("Wigan",))}
+    failures = pre_transformation_check(unauthorised_failure_dict)
+    assert len(failures) == 1
+    assert isinstance(failures[0], vf.UnauthorisedSubmissionFailure)
```

Comment:
can we also have a test for an authorized submission

Reply:
That's effectively what this is:

```
def test_extract_round_four_submission_details(valid_workbook_round_four):
    details_dict = extract_submission_details(valid_workbook_round_four, 4, place_names=("Newark",))

    assert details_dict["Form Version"] == (
        "Town Deals and Future High Streets Fund Reporting Template (v4.0)",
        {"Town Deals and Future High Streets Fund Reporting Template (v4.0)"},
    )
    assert details_dict["Reporting Period"] == (
        "1 April 2023 to 30 September 2023",
        {"1 April 2023 to 30 September 2023"},
    )
    assert details_dict["Fund Type"] == (
        "Town_Deal",
        {"Town_Deal", "Future_High_Street_Fund"},
    )
    assert details_dict["Place Name"] == (
        "Newark",
        set(TF_PLACE_NAMES_TO_ORGANISATIONS.keys()),
    )
 ```
 
 places_names contains Newark, the worksheet has Newark selected as the place_name